i18n/fr/docusaurus-plugin-content-docs/version-23.10/graph-views/install-ha.md

```diff
@@ -0,0 +1,797 @@
+| MAP Servers + QDevice | MAP Servers + QDevice | Corosync | UDP 5404 | Communication à l'intérieur du cluster (Multicast)                                |
+| MAP Servers + QDevice | MAP Servers + QDevice | Corosync | UDP 5405 | Communication à l'intérieur du cluster (Unicast)                                  |
+| MAP Servers + QDevice | MAP Servers + QDevice | PCS      | TCP 2224 | Communication à l'intérieur du cluster                                            |
+| MAP Servers + QDevice | MAP Servers + QDevice | Corosync | TCP 5403 | Communication avec le QDevice                                                     |
+
+</TabItem>
+<TabItem value="4 nœuds" label="4 nœuds">
+
+| De                         | Destination                 | Protocol | Port     | Application                                                                       |
+| :------------------------- | :-------------------------- | :------- | :------- | :-------------------------------------------------------------------------------- |
+| Nœud Base de données actif | Nœud Base de données passif | MariaDB    | TCP 3306 | Synchronisation MariaDB (doit être également ouvert du nœud passif vers le nœud actif) |
+| MAP Servers + DB + QDevice | MAP Servers + DB + QDevice  | Corosync | UDP 5404 | Communication à l'intérieur du cluster (Multicast)                                |
+| MAP Servers + DB + QDevice | MAP Servers + DB + QDevice  | Corosync | UDP 5405 | Communication à l'intérieur du cluster (Unicast)                                  |
+| MAP Servers + DB + QDevice | MAP Servers + DB + QDevice  | PCS      | TCP 2224 | Communication à l'intérieur du cluster                                            |
+| MAP Servers + DB + QDevice | MAP Servers + DB + QDevice  | Corosync | TCP 5403 | Communication avec le QDevice                                                     |
+
+</TabItem>
+</Tabs>
+
+### Installation de la plateforme Centreon MAP
+
+Un cluster Centreon MAP HA ne peut être installé que sur la base d'une plateforme Centreon MAP en fonctionnement.
+Avant de suivre cette procédure, il est nécessaire que **[cette procédure d'installation](map-web-install.md)** ait déjà été complétée et qu'**au moins 5GB d'espace libre soit disponible dans le volume groupe LVM** qui contient le répertoire de données MariaDB (point de montage `/var/lib/mysql` par défaut).
+
+La réponse de la commande `vgs` doit ressembler à ceci (bien vérifier la valeur sous la mention `VFree`) :
+
+```bash
+  VG      #PV #LV #SN Attr   VSize  VFree
+  vg_data   1   1   0 wz--n- 10,99g 5,99g
+  vg_root   1   2   0 wz--n-  9,00g    0 
+```
+
+Les deux serveurs Centreon MAP doivent être liés au même serveur central.
+Le script `/etc/centreon-map/diagnostic.sh` doit retourner `[OK]` sur **les deux** serveurs MAP :
+
+`Note: bien penser à updater les output`
```

Comment:
J'ai bien vu ta note ! :)
Le nouvel output : 

```suggestion
```bash
########## Centreon Map server version ##########

  [INFO] ii centreon-map-engine 23.10.3-bullseye amd64 Centreon Map service under Spring Boot framework

########## System ##########

  [INFO] SELinux is not available
Unit firewalld.service could not be found.
  [OK]   Firewall is disabled
  [INFO] Physical memory available on the server: 4013532 kb.
  [INFO] Number of CPU available on the server: 2 core(s)

########## Java ##########

  [OK]   Java 17 installed
  [OK]   Optimization found for JVM: JAVA_OPTS="-Xms512m -Xmx3G -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/var/log/centreon-map -Dmanagement.endpoints.enabled-by-default=false -Dmanagement.endpoint.health.enabled=true -Dmanagement.endpoint.metrics.enabled=true"

########## Database connection ##########

  [OK]   Connection to centreon
  [OK]   Connection to centreon_storage
  [OK]   Connection to centreon_map

########## Security ##########

  [OK]   Token signing key

########## Broker connection ##########

  [OK]   Connection to @CENTRAL_IPADDR@ 5758 port

########## Authentication ##########

  [OK]   Centreon Central authentication using user centreon_map

########## Protocol verification ##########

    [OK] Centreon Map server configured to use HTTPS protocol
    [INFO] Centreon Central configured in Map to use https protocol.
    [OK]   Centreon Central successfully answered to HTTPS request
  ```